redun/cli.py

```diff
@@ -2706,13 +2706,8 @@ def oneshot_command(self, args: Namespace, extra_args: List[str], argv: List[str
         """
         array_job_index: int = -1
         if args.array_job:
-            if AWS_ARRAY_VAR in os.environ:
-                array_job_index = int(os.environ[AWS_ARRAY_VAR])
-            elif K8S_ARRAY_VAR in os.environ:
-                array_job_index = int(os.environ[K8S_ARRAY_VAR])
-            elif GCP_ARRAY_VAR in os.environ:
-                array_job_index = int(os.environ[GCP_ARRAY_VAR])
-            else:
+            array_job_index = get_job_array_index()
```

Comment:
Small refactoring to isolate this logic, since now we have several executors that have an array notion.